page-full-width.php

```diff
@@ -52,6 +52,13 @@
         ?>
         </div>
     <?php
+    } elseif (is_front_page() !== false) {
```

Comment:
Could simplify to elseif (is_front_page()) as function only returns true or false

Reply:
Changed 👍